### PR TITLE
Fix build.sh wrong args in README.md

### PR DESCRIPTION
**Pull request by tcodehuber**
## Why are the changes needed?

"./build.sh ams" should be "./build.sh amoro" for refactoring code

## Brief change log

-

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ You can build all image via script in current dir.
 or just build only one image.
 
 ```shell
-./build.sh ams
+./build.sh amoro
 ```
 
 - NOTICE: The ams image and flink image required the project had been packaged.
```
Comment (HuangFru) [inline docker/README.md:34]: There are some old instructions in the document. Can you change them together?
This should be 'amoro image and quickdemo image required ....'.

**Review by HuangFru — CHANGES_REQUESTED** [review]
Thanks for your contribution. In addition to this, there are some old instructions in the document. Can you change them together?
Such as '

**Comment by tcodehuber** [comment]
> Thanks for your contribution. In addition to this, there are some old instructions in the document. Can you change them together?

OK

File: docker/README.md (hunk 1 above)
Comment (tcodehuber) [reply docker/README.md]: > This should be 'amoro image and quickdemo image required ....'.

Please review it again ,thx

Comment (HuangFru) [reply docker/README.md]: The quickdemo image also required the project had been packaged.

Comment (tcodehuber) [reply docker/README.md]: > The quickdemo image also required the project had been packaged.

done

**Review by HuangFru — APPROVED** [review]
LGTM.

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**